test/simulator/timestamp/src/include/timestamp_manager.h

```diff
@@ -48,7 +48,7 @@ class timestamp_manager {
     int validate_oldest_and_stable_ts(uint64_t &, uint64_t &, bool &, bool &);
     int validate_durable_ts(const uint64_t &, const bool &) const;
     int validate_read_timestamp(session_simulator *, const uint64_t) const;
-    int validate_commit_timestamp(session_simulator *, const uint64_t) const;
+    int validate_commit_timestamp(session_simulator *, uint64_t);
```

Comment:
Did you need to remove the last `const`? Is it modifying any members now?

Reply:
Yes, thats right.